client_start_charge.go

```diff
@@ -0,0 +1,73 @@
+package monta
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+)
+
+// StartChargePointsRequest is the request input to the [Client.StartChargePoints] method.
+type StartChargePointsRequest struct {
+	// The id of the team that will be paying for the charge.
+	PayingTeamID int64 `json:"payingTeamId"`
+	// Id of the charge point used for this charge.
+	ChargePointID int64 `json:"chargePointId"`
```

Comment:
GoDoc comments should contain the name of the thing they are commenting - in this case change to for example `ChargePointID is the ID of the charge point used for this charge.`

Reply:
Further reading: https://go.dev/doc/comment

Reply:
Please check the remainder of this PR for the same issue

Reply:
fixed!